platform-sdk/swirlds-platform-core/src/main/java/com/swirlds/platform/state/MerkleStateRoot.java

```diff
@@ -96,28 +93,16 @@
  * each child must be part of the state proof. It would be better to have a binary tree. We should
  * consider nesting service nodes in a MerkleMap, or some other such approach to get a binary tree.
  */
-public class MerkleHederaState extends PartialNaryMerkleInternal
-        implements MerkleInternal, SwirldState, HederaState, MerkleRoot {
-    private static final Logger logger = LogManager.getLogger(MerkleHederaState.class);
+@ConstructableIgnored
+public class MerkleStateRoot extends PartialNaryMerkleInternal
+        implements MerkleInternal, SwirldState, MerkleState, MerkleRoot {
+    private static final Logger logger = LogManager.getLogger(MerkleStateRoot.class);
 
     /**
      * Used when asked for a service's readable states that we don't have
      */
     private static final ReadableStates EMPTY_READABLE_STATES = new EmptyReadableStates();
 
-    // For serialization
-    /**
-     * This class ID is returned IF the default constructor was used. This is a nasty workaround for
-     * {@link ConstructableRegistry}. The registry does classpath scanning, and finds this class. It then invokes
-     * the default constructor and then asks for the class ID, and then throws away the object. It sticks this
-     * mapping of class ID to class name into a map. Later (in {@link Hedera}) we define an actual mapping for the
-     * {@link ConstructableRegistry} that maps {@link #CLASS_ID} to {@link MerkleHederaState} using a lambda that will
-     * create the instancing using the non-default constructor. But the {@link ConstructableRegistry} doesn't
-     * actually register the second mapping! So we will trick it. We will return this bogus class ID if the default
-     * constructor is used, or {@link #CLASS_ID} otherwise.
-     */
-    private static final long DO_NOT_USE_IN_REAL_LIFE_CLASS_ID = 0x0000deadbeef0000L;
-
     //    private static final long CLASS_ID = 0x2de3ead3caf06392L;
     // Uncomment the following class ID to run a mono -> modular state migration
     // NOTE: also change class ID of ServicesState
```

Comment:
Not a blocker, but these lines can be removed. They're irrelevant after the 0.49 (modularization) release.

Reply:
will do in the follow-up PR